frontend/lib/src/components/elements/Popover/styled-components.ts

```diff
@@ -17,5 +17,11 @@
 import styled from "@emotion/styled"
 
 export const StyledPopoverButtonIcon = styled.div(({ theme }) => ({
-  marginLeft: theme.spacing.twoXSPx,
+  marginLeft: theme.spacing.threeXSPx,
+  // This is a hacky way to offset the 5px "padding" of the expansion svg
+  // icon. Reason is that we want to use the same padding to the right side
+  // as the text on the left side. The alternative would be to overwrite the
+  // right padding of the button, which would also be hacky and involved slightly
+  // more logic.
+  marginRight: "-5px",
```

Comment:
I think it'd make sense to have these `-5px` as a variable for future re-use in case this is needed again. Also with a variable it might be easier to find it in case the padding changes in the future. If this is the only occurrence so far, just having the comment should be sufficient for now though although it could make sense to update your comment to reflect the thought in case someone will stumble across it in the future.

Reply:
Yep, I think it would make sense to somehow share this logic if we also want to apply it to another element. We probably want to apply a similar fix to the expander soon. I added additional comment here.